content/announcements/2024-12-10-contribfest-kubecon-eu-2025.md

```diff
@@ -0,0 +1,52 @@
+---
+slug: contribfest-kubecon-eu-2025
+title: Join Us for ContribFest at KubeCon EU 2025 in London!
+description: The cert-manager maintainers are hosting a ContribFest event on 2nd April, 2025, and you are invited!
+date: "2024-12-10T09:00:00Z"
```

Comment:
suggestion: might be worth updating this date (and the date in the filename)?

Reply:
Done, thanks.